.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug
+description: Report a problem with OpenDevin
+title: '[Bug]: '
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to fill out this bug report. We greatly appreciate your effort to complete this template fully. Please provide as much information as possible to help us understand and address the issue effectively.
+
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for the same bug?
+      description: Please check if an issue already exists for the bug you encountered.
+      options:
+      - label: I have checked the troubleshooting document at https://github.com/OpenDevin/OpenDevin/blob/main/docs/Troubleshooting.md
+        required: true
+      - label: I have checked the existing issues.
+        required: true
+
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: Provide a short description of the problem.
+    validations:
+      required: true
+
+  - type: textarea
+    id: current-version
+    attributes:
+      label: Current Version
+      description: What version are you using? If you're running in docker, tell us the tag you're using (e.g. ghcr.io/opendevin/opendevin:0.3.1).
+      render: bash
+    validations:
+      required: true
+
+  - type: textarea
+    id: config
+    attributes:
+      label: Installation and Configuration
+      description: Please provide the command you ran to start OpenDevin, and any configuration
```

Comment:
nit: the user might fail to install OpenDevin, so there's no "command you ran to start"